packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -210,7 +212,7 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
                 {onRenderNavigation(this.props, this._onRenderNavigation)}
               </div>
               <div className={_classNames.contentInner}>
-                {header}
+                {(isHeaderInContentInner || !hasCloseButton) && header}
```

Comment:
then this will read hasCustomNavigation or !hasCloseButton - which is more descriptive of the state. 

like why would we ever check hasCloseButton if we already have a variable stating if the header is in the content?

Reply:
Thank you for these explanations!